examples/cairo_snippets/snippets/glyph_path.py

```diff
@@ -10,13 +11,12 @@ def draw(cr, width, height):
     # draw 0.16 glyphs in 0.20 squares, at (0.02, 0.04) from left corner
     cr.set_font_size(0.16)
 
-    glyphs = []
```

Comment:
All not very beginner friendly imo.. but ymmv

Reply:
I agree. I don't think using the index with nested for loops is very pythonic, but I think the readability is better than using itertools. I dropped the commit. 👍